src/workloads/scale/ScanWithLongLived.yml

```diff
@@ -88,4 +88,5 @@ Actors:
 AutoRun:
   Requires:
     mongodb_setup:
-    - standalone
+    - single-replica
```

Comment:
Our workload is now running with read concern "snapshot" which doesn't work for standalone.
Let's just run against `single-replica` and without flow control.